### ISSUE TITLE
Please specify the license

### ISSUE THREAD

**Issue by prbinu** [snapcore/secboot#126]
This repo doesn't mention about any license. I would be helpful if you can specify the license (MIT, BSD, GPL etc.) for this code

**Comment by KhazAkar** [comment]
Yes, it's really important before anybody would use this code, or even look at it.
